build-etcher-riscv.sh

```
@@ -0,0 +1,884 @@
+timestamp()  { date '+%Y-%m-%d %H:%M:%S'; }
+
+echo "======================================================"
+echo "balenaEtcher RISC-V build — started $(timestamp)"
+echo "======================================================"
+echo ""
+
+# Work around V8 JIT bug on riscv64 (SpacemiT K1)
+# V8's JIT compiler generates invalid riscv64 code causing "unreachable code" crashes
+# --jitless forces interpreter-only mode: slower but stable
+export NODE_OPTIONS="--jitless"
+export MAKEFLAGS="-j$(nproc)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)